pyClient/zeth/joinsplit.py

```diff
@@ -493,17 +668,16 @@ def _encrypt_note(out_note: ZethNote, pub_key: EncryptionPublicKey) -> bytes:
 def receive_notes(
         addrs_and_ciphertexts: List[Tuple[int, bytes]],
         sender_k_pk: EncryptionPublicKey,
-        receiver_k_sk: EncryptionSecretKey) -> Iterable[Tuple[int, ZethNote]]:
+        receiver_k_sk: EncryptionSecretKey) -> Iterator[Tuple[int, ZethNote]]:
     """
     Given the receivers secret key, and the event data from a transaction
     (encrypted notes), decrypt any that are intended for the receiver.
     """
```

Comment:
Would you mind dropping a line to mention the structure of a note description returned by this function @dtebbs (basically what you said here https://github.com/clearmatics/zeth/pull/130#discussion_r359446403, a note desc being a note along with its address in the mktree)

Reply:
Added a description of the return values.